src/Nazara/VulkanRenderer/VulkanSurface.cpp

```diff
@@ -7,8 +7,17 @@
 #include <vulkan/vk_sdk_platform.h>
 #include <Nazara/VulkanRenderer/Debug.hpp>
 
+#ifdef VK_USE_PLATFORM_METAL_EXT
+#include <objc/runtime.h>
+#include <vulkan/vulkan_metal.h>
+#endif
+
 namespace Nz
 {
+	#ifdef VK_USE_PLATFORM_METAL_EXT
+	id CreateAndAttachMetalLayer(void *window);
```

Comment:
Please use `void* window` instead of `void *window`

Reply:
I should have fixed the cases with `type *` and `type &` now.